Watchman.Configuration/Generic/IAwsServiceAlarms.cs

```diff
@@ -0,0 +1,13 @@
+using System.Collections.Generic;
+
+namespace Watchman.Configuration.Generic
+{
+    public interface IAwsServiceAlarms
```

Comment:
non-generic interface added for consumers who don't care about the custom configuration that is `TResourceConfig`